### ISSUE TITLE
Feature request: Add a "contacts" param to the campaign' send API

### ISSUE THREAD

**Issue by lexoyo** [pentacent/keila#344]
Hello

I'm basically trying to use keila to send transactional emails

My idea is to use the api with n8n.io or something similar. If I had just a param "contacts" in the campaign' send API, I could achieve that right?

**Comment by wmnnd** [comment]
Hey @lexoyo, thanks for suggesting this. Transactional emails have been on my personal to-do list for Keila for a while. 

Before we can implement them, we need to do a somewhat big refactoring as described in #355.

**Comment by filipesmedeiros** [comment]
Hey! Thank you so much for the work. I hope this isn't toxic in any way, but I was wondering if you have a predicted timeline on this? No demands, just a ballpark, if you have one. I would love to pitch Keila to my organisation (a non-profit). :)

**Comment by wmnnd** [comment]
#355 is currently one of the top priorities on the roadmap, after that sending transactional emails via the API will almost certainly be possible.

**Comment by icaria36** [comment]
Excuse my lack of technical knowledge. Would "transactional emails" support allow us to use Keila instead of i.e. Mailgun as an SMTP provider? I'm considering moving away from mailgun for my couple of Discourse-based projects -- see https://github.com/discourse/discourse/blob/main/docs/INSTALL-email.md.

**Comment by wmnnd** [comment]
Yes, more or less – though at first this would be an API feature and not be available via the regular SMTP protocol.

**Comment by Phobetor** [comment]
@wmnnd According to [this comment](https://github.com/pentacent/keila/issues/355#issuecomment-4789530272), transactional email is available. Is this issue obsolete or is it somehow different?

**Comment by wmnnd** [comment]
Oh yes, thanks for pointing that out, transactional emails are available already.
